conf/livy.conf

```diff
@@ -38,3 +38,7 @@
 # default it's empty, meaning users can only reference remote URIs when starting their
 # sessions.
 # livy.file.local-dir-whitelist =
+
+# Whether to enable csrf protection, by default it is false. If it is enabled, client should add
+# http-header "X-Requested-By" in request if the http method is POST/DELETE/PATCH.
```

Comment:
nit: s/PATCH/PUT (I don't think we have any "patch" endpoints). I'll fix it during merge.